packages/api/ecs/async-operation/README.md

```diff
@@ -2,15 +2,15 @@
 
 This image is intended for use with the AsyncOperation model. It will download
 an AWS Lambda function from AWS, download a payload from S3, execute the lambda
-function (inside the Docker container), then update the DynamoDB record for the
+function (inside the Docker container), then update the RDS database record for the
 AsyncOperation with the result of the lambda function.
 
 The Docker container expects a number of environment variables to be set:
 
 * lambdaName - the name of the lambda function to be executed
 * payloadUrl - an S3 URL containing the event to be passed to the lambda
   function
-* asyncOperationsTable - the name of the AsyncOperations DynamoDB table
+* asyncOperationsTable - the name of the AsyncOperations RDS database table
```

Comment:
This variable actually does still refer to the DynamoDB table with parallel writes. It's not necessary for the RDS writes

Reply:
Ok see if this is better e41f234dd82f48333f3c14de8fa8710e9d93b81e